src/sdk/models/policies.ts

```diff
@@ -83,7 +91,7 @@ export interface PolicyTrackingGroupRule extends IEntityIdentifier<string> {
 	criteria?: MetadataCalculatedFilter;
 }
 
-interface PolicyActionBase extends IEntityIdentifier<string> {
+interface PolicyActionBase extends IEntityIdentifier<PolicyActionTypeID> {
```

Comment:
Shouldn't this still allow custom ids? @smbecker thoughts?
```suggestion
interface PolicyActionBase extends IEntityIdentifier<PolicyActionTypeID | string> {
```

Reply:
I agree with @gotvino on this one, because it self-documents the relationship, while still allowing for a `string` fallback.

Reply:
Okay, it has been added back.